Source/Core/VideoCommon/VideoConfig.h

```diff
@@ -51,6 +51,14 @@ enum StereoMode
 	STEREO_3DVISION
 };
 
+constexpr int STEREOSCOPY_PRESETS_NUM = 3;
```

Comment:
Nice idea. But why exactly 3? We can also just store a vector :)

But so the hotkeys would likely be HK_USE_STEREOSCOPY_PRESET_INC and HK_USE_STEREOSCOPY_PRESET_DEC.

Feel free to ignore this comment.